azure/packages/azure-client/src/AzureAudience.ts

```diff
@@ -26,14 +27,21 @@ export class AzureAudience extends ServiceAudience<AzureMember> implements IAzur
 	 * @internal
 	 */
 	protected createServiceMember(audienceMember: IClient): AzureMember {
-		const azureUser = audienceMember.user as AzureUser;
-		assert(azureUser?.name !== undefined, 'Provided user was not an "AzureUser".');
-
-		return {
-			userId: audienceMember.user.id,
-			userName: azureUser.name,
-			connections: [],
-			additionalDetails: azureUser.additionalDetails as unknown,
-		};
+		return createAzureAudienceMember(audienceMember);
 	}
 }
+
+/**
+ * Creates Azure-specific audience member
+ */
+export function createAzureAudienceMember(audienceMember: IClient): AzureMember {
+	const azureUser = audienceMember.user as AzureUser;
+	assert(azureUser?.name !== undefined, 'Provided user was not an "AzureUser".');
```

Comment:
Should we do `typeof azureUser?.name === "string"` here instead? IIRC `userName` is typed as `string`.

Reply:
this is just copy pasted from the current implementation, i didn't change it, and don't want to in this pr